src/utils/next.ts

```diff
@@ -210,6 +210,7 @@ export const scaffold =
           INSTANCE_OWNER_HREF: process.env.INSTANCE_OWNER_HREF,
           INSTANCE_OWNER_NAME: process.env.INSTANCE_OWNER_NAME,
           MUIX_LICENSE_KEY: process.env.MUIX_LICENSE_KEY,
+          TILESERVER: process.env.TILESERVER,
```

Comment:
I think we should add a fallback here.

```suggestion
          TILESERVER: process.env.TILESERVER || 'https://tile.openstreetmap.org',
```